tests/migration_test.go

```diff
@@ -1083,7 +1083,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				tests.UpdateClusterConfigValueAndWait("migrations", originalMigrationConfig)
 			})
 			It("[test_id:2227]should abort a vmi migration without progress", func() {
-				tests.SkipIpv6TestsFailingAfterVmImageUpgradeToFc32()
+				tests.SkipMigrationTestIfRunnigOnKindInfra()
```

Comment:
this should be here please, not migration
				tests.SkipStressTestIfRunnigOnKindInfra()

Reply:
it need just stressTest, but i understand what was confusing me.
my PR adds the stress Skip
in your it isnt available yet, so its ok that you put migration, sorry for the noise

here my PR adds the stress skip
https://github.com/kubevirt/kubevirt/pull/3392/files#diff-a5a178b4c66835110ceff63c61d28c96R4664
it was groomed and researched as part of my PR

Reply:
Ok, now it is clear.
So if your PR gets in first, then on rebase of this one the new skip should appear.
And as `SkipMigrationTestIfRunnigOnKindInfra` will be gone, we will not forget it for sure.